include/fem-dem/parameters_cfd_dem.h

```diff
@@ -88,6 +89,7 @@ namespace Parameters
     double                         l2_smoothing_factor;
     double                         l2_lower_bound;
     double                         l2_upper_bound;
+    unsigned int                   number_of_particle_refinement;
```

Comment:
If you decide to change the name of the parameter, I think it is a good idea to do it here too, to give a more meaningful name to it.